packages/tldraw/src/lib/ui/components/A11y.tsx

```diff
@@ -0,0 +1,26 @@
+import { useValue } from '@tldraw/editor'
+import { memo } from 'react'
+import { useA11y } from '../context/a11y'
+
+/** @public @react */
+export const DefaultA11y = memo(function TldrawUiA11y() {
```

Comment:
We may add other a11y related components in the future, so I'd like to give this a slightly more specific name - `A11yAnnouncer`, perhaps?